dev-tools/mage/manifest/manifest.go

```diff
@@ -110,6 +110,16 @@ func DownloadComponentsFromManifest(manifest string, platforms []string, platfor
 	// This eliminates the "+buildYYYYMMDDHHMM" suffix on Independent Agent Release builds
 	majorMinorPatchVersion := parsedManifestVersion.VersionWithPrerelease()
 
+	// For Independent Agent Releases, any opted-in projects will have a version that is
+	// one Patch version ahead since we are using the new bumped DRA artifacts for those projects.
+	// This is acceptable since it is being released only as bundled with Elastic Agent.
+	patchPlusOneString := fmt.Sprintf("%d.%d.%d", parsedManifestVersion.Major(), parsedManifestVersion.Minor(), parsedManifestVersion.Patch()+1)
```

Comment:
Is there a reason why the input manifest can't specify the patch+1 version? That would keep the logic for this in the release jobs, with the agent package target just doing what the manifest tells it to.

Reply:
that may be a decent way to go.

I think the way it works is that it uses the `version` value in the header, which is something like `8.13.4+build202406101001`.  And then we strip off the metadata to get the base 3-number version for the previously-released (opted-out) projects.

We could probably somewhat easily add something like an `optin_version` in the header, but that doesn't tell us exactly _which_ projects should use that version.

As it is now, we'd probably have to go through and parse all of the packages in each of the projects and find any that have a filename that includes the `patch+1` version in it.

Perhaps we could also include just a list of `opted_in_projects` at the top level?  

We probably need to ask @DaveSys911 whether adding fields to the Ind Agent manifest would be a big issue or not.